### PR TITLE
Add Office Add-ins rules.

### PR DESCRIPTION
**Pull request by jtothej**
Add Office Add-ins rules.

### CODE REVIEW

File: persistence/office/act-as-office-com-add-in.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    att&ck:
+      - Persistence::Office Application Startup::Add-ins [T1137.006]
+    references:
+      - https://labs.withsecure.com/publications/add-in-opportunities-for-office-persistence
+      - https://learn.microsoft.com/en-us/dotnet/api/extensibility.idtextensibility2?view=visualstudiosdk-2022
+    examples:
+      - 0831bb382211a67c57a392955138808526aa15e55531091841706aae2cb89613
+  features:
+    - and:
+      - class: "Extensibility.IDTExtensibility2"
```
Comment (williballenthin) [inline persistence/office/act-as-office-com-add-in.yml:17]: is this a .NET class? perhaps we should also add a `format: dotnet` feature to make it clear

File: persistence/office/act-as-word-wll-add-in.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    scope: file
+    att&ck:
+      - Persistence::Office Application Startup::Add-ins [T1137.006]
+    references:
+      - https://www.ired.team/offensive-security/persistence/word-library-add-ins
+    examples:
+      - 03bb32d43885e83bc56c0b2bcad6f0c5ea40402763b7057056c654990022471b
+  features:
+    - or:
+      - export: wdAutoOpen
```
Comment (williballenthin) [inline persistence/office/act-as-word-wll-add-in.yml:16]: very cool

**Review by williballenthin — APPROVED** [review]
one optional suggestion, looks great!

File: persistence/office/act-as-office-com-add-in.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      - Persistence::Office Application Startup::Add-ins [T1137.006]
+    references:
+      - https://labs.withsecure.com/publications/add-in-opportunities-for-office-persistence
+      - https://learn.microsoft.com/en-us/dotnet/api/extensibility.idtextensibility2?view=visualstudiosdk-2022
+    examples:
+      - 0831bb382211a67c57a392955138808526aa15e55531091841706aae2cb89613
+  features:
+    - and:
+      - format: dotnet
+      - class: "Extensibility.IDTExtensibility2"
```
Comment (mr-tz) [inline persistence/office/act-as-office-com-add-in.yml:18]:
```suggestion
      - class: Extensibility.IDTExtensibility2
```

Comment (mr-tz) [reply persistence/office/act-as-office-com-add-in.yml:18]: Or are the quotes required?

Comment (jtothej) [reply persistence/office/act-as-office-com-add-in.yml]: Thanks! Removed quotes in https://github.com/mandiant/capa-rules/pull/781/commits/e8b50a13847ca6c8a5e7b668c113489b6cf476de

**Comment by mr-tz** [comment]
Thank you!

**Pull request merged**